prime-router/src/main/kotlin/azure/WorkflowEngine.kt

```diff
@@ -97,9 +97,17 @@ class WorkflowEngine(
         report: Report,
         actionHistory: ActionHistory,
         receiver: Receiver,
-        txn: Configuration? = null
+        txn: Configuration? = null,
+        context: ExecutionContext? = null
     ) {
-        val blobInfo = blob.uploadBody(report)
+        val blobInfo = try {
+            // formatting errors can occur down in here.
+            blob.uploadBody(report)
+        } catch (ex: Exception) {
+            context?.logger?.warning("Got exception while dispatching to schema ${report.schema.name}" +
+                ", and rcvr ${receiver.fullName}")
+            throw ex
+        }
```

Comment:
Added this logging because low level errors were occurring down in the bowels of the Element.kt, but it was impossible to tell which schema/receiver the errors were for.

It feels odd to me that the formatting of data for export occurs down inside a method called 'blob.uploadBody'.